### ISSUE TITLE
ms-NavBar has low to white contrast ratio so difficult to notice

### ISSUE THREAD

**Issue by knutkj** [OfficeDev/Office-UI-Fabric#477]
`ms-NavBar` has low to white contrast ratio ([`1.1`](http://leaverou.github.io/contrast-ratio/#%23f4f4f4-on-white)) so that it is difficult to notice it on a page with a white background. It should have more color options like Bootstraps's Navbar:

http://getbootstrap.com/components/#navbar-inverted

Would love one with [Theme Darker `#004578`](http://dev.office.com/fabric/styles#color).

**Comment by mikewheaton** [comment]
I'll pass this feedback on to our design team. For now this is the only color that the NavBar comes in, but you're welcome to write CSS overrides to change it's color. For example:

```
.ms-NavBar {
  background-color: #004578;
}

.ms-NavBar-link {
  color: #ffffff;
}
```
